chain/jsonrpc/res/sync_info.html

```diff
@@ -37,20 +38,65 @@
+                        if (validatorSet.has(element.account_id)) {
+                            if (element.next_hops != null && element.next_hops.length > 0) {
+                                reachableSet.add(element.account_id);
+                            }
+                        }
+                    });
+
+                    $('.js-num-validators').text(data.validators.length);
+                    $('.js-num-known-validators').text(knownSet.size);
+                    $('.js-num-reachable-validators').text(reachableSet.size);
+
                     let sync_status = data.detailed_debug_status.sync_status;
                     let network_info = data.detailed_debug_status.network_info;
                     $('.js-sync-status').text(sync_status);
                     $('.js-max-peers').text(network_info.peer_max_count);
                     $('.js-num-peers').text(network_info.num_connected_peers);
-                    network_info.connected_peers.forEach(function(peer, index) {
-                        $('.js-tbody-peers').append($('<tr>')
-                            .append($('<td>').append(peer.addr))
-                            .append($('<td>').append(JSON.stringify(peer.account_id)))
-                            .append($('<td>').append(JSON.stringify(peer.height)))
-                            .append($('<td>').append(JSON.stringify(peer.tracked_shards)))
-                            .append($('<td>').append(JSON.stringify(peer.archival)))
-                        )
+                    network_info.connected_peers.forEach(function (peer, index) {
+                        let peer_id = peer.peer_id;
+                        let validator = new Array();
+                        let routedValidator = new Array();
+                        data.detailed_debug_status.network_info.known_producers.forEach(element => {
+                            if (validatorSet.has(element.account_id)) {
+                                if (element.peer_id == peer_id) {
+                                    validator.push(element.account_id);
+                                } else {
+                                    if (element.next_hops != null) {
+                                        if (element.next_hops.includes(peer_id)) {
+                                            routedValidator.push(element.account_id);
+                                        }
+                                    }
+                                }
+                            }
+                        });
+                        let row =
+                            $('.js-tbody-peers').append($('<tr>')
+                                .append($('<td>').append(peer.addr))
+                                .append($('<td>').append(validator.join(",")))
+                                .append($('<td>').append(JSON.stringify(peer.account_id) + " " + peer.peer_id.substr(9, 5) + "..."))
```

Comment:
It seems to me that the first part `JSON.stringify(peer.account_id)` is all `null` on http://34.147.91.25:3030/debug/sync_info. Is that intended?